engine/src/main/java/org/hibernate/search/bridge/spi/FieldType.java

```diff
@@ -48,5 +48,10 @@
 	/**
 	 * A double field.
 	 */
-	DOUBLE
+	DOUBLE,
+
+	/**
+	 * A container for othe fields, for example an embedded object or a map.
```

Comment:
"othe"="other".

I haven't fully groked yet why this is needed now. Could you explain a bit?